sunpy/util/parfive_helpers.py

```diff
@@ -4,27 +4,49 @@
 
 import aiohttp
 import parfive
+from packaging.version import Version
 from parfive import Results
 
 import sunpy
 
-__all__ = ['Downloader', 'Results']
```

Comment:
To avoid making lots of changes, I think we should just pin the major version of parfive to `<2` if it's not compatible with `3.1.x`.